client/src/components/App/Content/InvestigationForm/TabManagement/InteractionsTab/ContactDateCard/ContactDateCard.tsx

```diff
@@ -63,6 +64,7 @@ const ContactDateCard: React.FC<Props> = (props: Props) => {
 }
 
 interface Props {
+    allInteractions: Interaction[];
```

Comment:
I wonder why moving it as prop and not creating context to the 4th and 5th tab?

Reply:
I agree
For now it doesn't matter